dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -203,7 +205,7 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
   end
 
   test "leave with valid joined section code" do
```

Comment:
Should we also add a test to verify that a student can't leave a section?

Reply:
Yes definitely 👍 . It's actually just leaving the test in that I updated and expecting a 403 instead. Will add that now